src/modules/Login/action.js

```diff
@@ -193,8 +194,8 @@ export const mergeSettings = (
   defaults: Object,
   types: Object,
   account?: Object
-): { finalSettings: Object, isOverwriteNeeded: boolean, isDefaultTypeIncorrect: boolean } => {
-  const finalSettings = {}
+): { finalSettings: AccountInitPayload, isOverwriteNeeded: boolean, isDefaultTypeIncorrect: boolean } => {
+  const finalSettings: any = {}
```

Comment:
just want to confirm that we can't use special type here `AccountInitPayload | {}` or something